staging/src/k8s.io/cloud-provider/controllers/service/controller.go

```diff
@@ -678,6 +680,13 @@ func nodeNames(nodes []*v1.Node) sets.String {
 	return ret
 }
 
+func loggableNodeNames(nodes []*v1.Node) []string {
```

Comment:
let's use a different PR for the logging problem to not conflate two different things, we also need to backport the bugfix

Reply:
sorry, I mean different commits ... for context for other reviewers, this is problematic on 5K, 10K and 15K clusters